blues/simulation.py

```diff
@@ -226,35 +215,33 @@ def simulateNCMC(self, verbose=False, write_ncmc=False):
         """Function that performs the NCMC simulation."""
         #append nc reporter at the first step
         if (self.current_iter == 0) and (write_ncmc):
-            self.ncmc_reporter = app.dcdreporter.DCDReporter(self.ncmc_outfile, 1) 
+            self.ncmc_reporter = app.dcdreporter.DCDReporter(self.ncmc_outfile, 1)
             self.nc_sim.reporters.append(self.ncmc_reporter)
         for nc_step in range(self.nstepsNC):
             try:
                 self.current_stepNC = int(nc_step)
                 # Calculate Work/Energies Before Step
-                work_initial = self.getWorkInfo(self.nc_integrator, self.work_keys)
-                     
+                if verbose:
+                    work_initial = self.getWorkInfo(self.nc_integrator, self.work_keys)
 
                 # Attempt NCMC Move
-                if int(self.mover.moves['step']) == nc_step:
-                    print('[Iter {}] Performing NCMC {} move'.format(
-                    self.current_iter, self.mover.moves['method'].__name__))
+                if self.nstepsNC / 2 == nc_step:
```

Comment:
I have a little bit of an issue with following this bit of code -- maybe comment slightly more carefully? Specifically, I'm just baffled by the factor of 2. I'm assuming this is because of something having to do with the total number of steps needing to be divisible by 2...